ansible/README.md

```diff
@@ -310,3 +310,10 @@ ansible-inventory -i inventory.yml --graph
 
 ansible servers -m ping -u ubuntu -i inventory.yml
 ```
+
+#### Encoding single variables
+
+```terminal
+# Notice the leading space
+ ansible-vault encrypt_string '000.000.000.000' --name ansible_host >> host_vars/server.yml
+```
```

Comment:
Interesting, I just use `ansible-vault edit` for everything

Reply:
Oh I see. I personally prefer only file-level encryption, but I don't mind doing it this way either. Thanks for documenting it in any case.

Reply:
Changed the directory structure of `host_vars`